langchain/chains/query_constructor/query_language.py

```diff
@@ -0,0 +1,72 @@
+from abc import ABC, abstractmethod
```

Comment:
alternative names for file name is `ir.py` (aka internal representation or a more verbose version) or `ast.py`  (in case you like any of these)